stellar/deploy-contract.js

```diff
@@ -96,36 +100,82 @@ async function processCommand(options, config, chain) {
     await broadcast(operation, wallet, chain, 'Initialized contract', options);
 }
 
-async function mainProcessor(options, processor) {
-    const config = loadConfig(options.env);
-    await processor(options, config, getChainConfig(config, options.chainName));
-    saveConfig(config, options.env);
-}
+async function upgrade(options, _, chain, ..._args) {
+    const { privateKey, wasmPath, yes } = options;
+    const { rpc, networkType } = chain;
+    const networkPassphrase = getNetworkPassphrase(networkType);
+    const contractAddress = chain.contracts?.axelar_gateway.address;
```

Comment:
This should be allowed for any contract and not just the gateway (this is a generic deploy script).

Reply:
got it. made it generic